src/lat/lattice-functions.h

```diff
@@ -28,14 +28,24 @@
 #include <map>
 
 #include "base/kaldi-common.h"
-#include "hmm/posterior.h"
 #include "fstext/fstext-lib.h"
-#include "hmm/transition-model.h"
 #include "lat/kaldi-lattice.h"
 #include "itf/decodable-itf.h"
+#include "itf/transition-information.h"
+#ifndef KALDI_MINIMAL_LATTICE_FUNCTIONS
+#include "lat/lattice-functions-transition-model.h"
+#endif // KALDI_MINIMAL_LATTICE_FUNCTIONS
```

Comment:
I thought we were getting rid of `KALDI_MINIMAL_LATTICE_FUNCTIONS`? There are two separate headers and two .cc files.
```suggestion
#include "lat/lattice-functions-transition-model.h"
```

Please also add
```c++
#include "lat/lattice-functions.h"
```
the first thing in the file while you're there--I think it's missing entirely